From f533547c56948c9512b928e91708d3d01128d68d Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Thu, 8 Aug 2024 02:23:48 +0800
Subject: [PATCH 07/26] iwyu: include cstddef for nullptr_t in
 fragment_data_iterator.h

Bug: 41455655
Change-Id: I7ad70dc8b2b005d75baf0b3d0578e561ca4864fa
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5426194
Reviewed-by: Xianzhu Wang <wangxianzhu@chromium.org>
Reviewed-by: Morten Stenshorne <mstensho@chromium.org>
Commit-Queue: Xianzhu Wang <wangxianzhu@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1321762}
Co-authored-by: lauren n. liberda <lauren@selfisekai.rocks>
---
 third_party/blink/renderer/core/paint/fragment_data_iterator.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/third_party/blink/renderer/core/paint/fragment_data_iterator.h b/third_party/blink/renderer/core/paint/fragment_data_iterator.h
index 8c42bce8f2..24e4a64936 100644
--- a/third_party/blink/renderer/core/paint/fragment_data_iterator.h
+++ b/third_party/blink/renderer/core/paint/fragment_data_iterator.h
@@ -5,6 +5,7 @@
 #ifndef THIRD_PARTY_BLINK_RENDERER_CORE_PAINT_FRAGMENT_DATA_ITERATOR_H_
 #define THIRD_PARTY_BLINK_RENDERER_CORE_PAINT_FRAGMENT_DATA_ITERATOR_H_
 
+#include <cstddef>
 #include <optional>
 
 #include "third_party/blink/renderer/core/layout/inline/inline_cursor.h"
@@ -22,7 +23,7 @@ class FragmentDataIteratorBase {
 
  public:
   explicit FragmentDataIteratorBase(Head& head) : fragment_head_(head) {}
-  explicit FragmentDataIteratorBase(nullptr_t) {}
+  explicit FragmentDataIteratorBase(std::nullptr_t) {}
 
   Data* GetFragmentData() const {
     return !IsDone() ? &fragment_head_.at(idx_) : nullptr;
-- 
2.46.0

